sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -72,57 +72,76 @@ case class CurrentTimestamp() extends LeafExpression with CodegenFallback {
 }
 
 /**
- * Adds a number of days to startdate.
+ * Adds a number of days to date/timestamp.
  */
 @ExpressionDescription(
-  usage = "_FUNC_(start_date, num_days) - Returns the date that is num_days after start_date.",
+  usage = "_FUNC_(instant, num_days) - Returns the date/timestamp that is num_days after instant.",
   extended = "> SELECT _FUNC_('2016-07-30', 1);\n '2016-07-31'")
-case class DateAdd(startDate: Expression, days: Expression)
+case class DateAdd(instant: Expression, days: Expression)
```

Comment:
Maybe to follow the function name `date_add`?  It seems it just followed hive though, [GenericUDFDateAdd.java](https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDFDateAdd.java). I was actually wondering if `TruncDate` is appropriate name too because it seens it does not follow the function name `trunc` nor hive, [GenericUDFTrunc.java](https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDFTrunc.java).